code/modules/shuttle/shuttle.dm

```diff
@@ -163,10 +165,8 @@
 /obj/docking_port/stationary/Initialize(mapload)
 	. = ..()
 	SSshuttle.stationary += src
-	if(!id)
-		id = "[SSshuttle.stationary.len]"
-	if(name == "dock")
-		name = "dock[SSshuttle.stationary.len]"
+	id = "[SSshuttle.stationary.len]"
+	name = "[name] dock[SSshuttle.stationary.len]"
```

Comment:
Do not remove the previous code, this is to handle map edits

Reply:
how to handle !00% different ID and NAME near shuttles in another way?

Reply:
this code want change id if we have 2 or more of dock/shuttles but i dont see it work. so i remove the check and give unical suffix

Reply:
I don't understand what you're saying here

Reply:
Problem in map edited shuttles, it ID and NAME same for all created by template.

Reply:
Then remove those map edits causing problems and being used improperly. That's not a reason to remove map edit support

Reply:
I return map edit support